src/components/CreateHabit.js

```diff
@@ -0,0 +1,84 @@
+import React, { Component } from 'react';
+import { StyleSheet } from 'react-native';
+import { Button, Dialog, TextInput, Portal, Surface } from 'react-native-paper';
+//REDUX
+import { connect } from 'react-redux';
+import { createHabit } from '../redux/actions';
+
+const styles = StyleSheet.create({
+    dialog: {
+        height: 400,
+        width: 325,
+    },
+    surface: {
+        padding: 8,
+        margin: 8,
+        marginLeft: 25,
+        height: 80,
+        width: 325,
+        alignItems: 'center',
+        justifyContent: 'center',
+        elevation: 4,
+    },
```

Comment:
A few things here on the styling. Is it possible for the widths be percentages? My iOS Simulator defaults to an iPhone 12 Pro Max and these elements are left-aligned and only use about 80% of the screen.

Additionally, it looks like some of the margins are different between the the `CreateHabit` component and the `LogButton` component. I think it's just that one is 20px and one is 25px, but two options pop to mind to ensure this is consistent in future:
- Setup a `styles` object which includes various spacing sizes and pull values from there (so something like `margin: styles.spacing.medium`)
- Wrap the elements in a component and apply padding to that parent component instead of applying margin to the children components (except for whatever margins you need to create space between the children components)

Reply:
Have added a theme to the App - very basic for now but has some universal 'spacing' for use in styling as suggested. Can build on this as project progresses.